src/main/java/com/minecolonies/apiimp/initializer/ModBuildingsInitializer.java

```diff
@@ -357,6 +357,7 @@ private ModBuildingsInitializer()
           .setBuildingProducer(BuildingTownHall::new)
           .setBuildingViewProducer(() -> BuildingTownHall.View::new)
           .setRegistryName(new ResourceLocation(Constants.MOD_ID, ModBuildings.TOWNHALL_ID))
+          .addBuildingModuleProducer(TOWNHALL_SETTINGS)
```

Comment:
Could you show an image of how the new settings list looks in the townhall

Reply:
![image](https://github.com/ldtteam/minecolonies/assets/38401808/7ab3de2f-f106-4e90-9907-6b6ac11f6235)